lms/djangoapps/instructor/tasks_helper.py

```diff
@@ -36,19 +36,29 @@ def get_section_users(course_id, query):
     return results
 
 
+def total_students_in_course(course_id):
```

Comment:
What this function is named and what the docstring says it does don't appear to be the same thing. Also, it's not clear to me that either (docstring or method name) actually represent what's happening here.

The docstring says that it returns IDs, but the code looks to be returning User objects.  The method name is called `total_students...` which implies that this function returns an integer/count, whereas it's really returning a collection.

For right now, I think we should remove this change from this PR, as it's already a bit outside the scope of this changeset. If you'd still like, we can look to implement this (addressing the above issues) in a separate PR.